app.py

```diff
@@ -71,7 +71,8 @@ def _has_signature_secret():
 
 # Create the application configuration
 certs_secret = os.path.join(SECRETS, 'streams-certs')
-if os.path.exists(certs_secret):
+server_pass = os.path.join(certs_secret, 'server.pass')
```

Comment:
This is exactly the same change as in #43. I added this change on my end before it was merged into master so I could work on endpoint_monitor.py